sonic_data_client/non_db_client.go

```diff
@@ -132,10 +132,13 @@ type cpuUtil struct {
 }
 
 func getCpuUtilPercents(cur, last *linuxproc.CPUStat) uint64 {
-	curTotal := (cur.User + cur.Nice + cur.System + cur.Idle + cur.IOWait + cur.IRQ + cur.SoftIRQ + cur.Steal + cur.Guest + cur.GuestNice)
-	lastTotal := (last.User + last.Nice + last.System + last.Idle + last.IOWait + last.IRQ + last.SoftIRQ + last.Steal + last.Guest + last.GuestNice)
+	curTotal := (cur.User + cur.Nice + cur.System + cur.Idle + cur.IOWait + cur.IRQ + cur.SoftIRQ + cur.Steal)
+	lastTotal := (last.User + last.Nice + last.System + last.Idle + last.IOWait + last.IRQ + last.SoftIRQ + last.Steal)
 	idleTicks := cur.Idle - last.Idle
 	totalTicks := curTotal - lastTotal
+	if totalTicks == 0 { // No change in CPU Utilization
+            return 0
+	}
```

Comment:
This is the fix for crash issue, do we really need to update pollStats()?

Reply:
I made the change as a way to be able to mock 3000 objects inside statsRBuffer so I can cover cases where we get cpu utilization up to the past 5 minutes. My previous change was to wait until pollStats added 3000 objects but that would require a 5 minute sleep in every build so I opted for this approach.

Reply:
Thx for the changes!